js/actions/index.js

```diff
@@ -4,7 +4,5 @@
  * @flow
  */
 
-'use strict'
```

Comment:
not needed anymore?

Reply:
Many of the strict rules are checked by the linter but others are uncovered. Most of the files are not declaring 'use strict'. If we decide to 'use strict' i think is better to add it in every file. What you think? 

NOTE: We can also add a danger rule to check that 'use strict' is included in every js file.

Reply:
there's [this](https://eslint.org/docs/rules/strict)